### DISCUSSION TITLE
Use my own geometry for 3D RBCs simulation

### DISCUSSION THREAD

**Discussion by Simosteinos** [UvaCsl/HemoCell#85] [General discussion]
Hello,
I want to use a specific geometry for simulationg RBCs in periodic conditions.
I tried drawing a 3D model in stl extension, and I tried to use it in the piplow example. but its not working.
Can someone please help me with that.
Thanks

**Comment by mojtabaAmiraslanpour** [comment]
I have been able to do this. What is the problem?

**Reply by Simosteinos** [reply]
Really?
Can you please explain me how you did it?

**Reply by mojtabaAmiraslanpour** [reply]
Just put the STL file in the directory and refer to it in the config file. If you describe the problem you are referring to, I can help you better.

**Reply by Simosteinos** [reply]
Indeed I put the STL file.
[Here ](https://www.mediafire.com/file/ueyof4ihfpel85z/Geo.stl/file) the file which I wanted to used.

**Reply by mojtabaAmiraslanpour** [reply]
As I said you can replace the STL file in the pipeflow case and change the name in the config file. Do you get any actual errors?
I would also suggest smoothing that STL geometry a bit.
